src/lib/network/network_autoyast.rb

```diff
@@ -227,6 +227,27 @@ def valid_rename_udev_rule?(rule)
       true
     end
 
+    def rename_lan_item(item, name_to, attr = nil, key = nil)
+      return if item.nil? || item.empty? || item < 0 || item >= LanItems.Items.size
```

Comment:
and that brings me to another big topic. I am missing test case for this method. I think it is doable.